android/app/src/main/java/org/fossasia/openevent/core/speaker/SpeakersListAdapter.java

```diff
@@ -137,4 +139,12 @@ else if (!TextUtils.isEmpty(speakerData) && (sortType == 1 || sortType == 2))
             holder.header.setText(String.valueOf("#"));
     }
 
+    @Override
+    public String getSectionTitle(int position) {
+        //this String will be shown in a bubble for specified position
+        Speaker speaker = getItem(position);
+        String name = speaker.getName();
+        return name.substring(0, 1);
+    }
```

Comment:
This is only valid for sorted by name, will break for other sort criterias

Reply:
I don't think so because the method executes for each item. So for each item, the first alphabet is taken and displayed.

Reply:
I tested it with other sort criteria. It works as expected.

Reply:
Not saying that. If the speakers are sorted by country, country's first character should be displayed in scroller and not name's

Reply:
oh okay. Ill make the changes

Reply:
@iamareebjamal For some speakers, there is no country or organization. What do you want me to do in that case?

Reply:
`-`

Reply:
done